ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/ListStreams.java

```diff
@@ -26,16 +26,26 @@
 public class ListStreams extends Statement {
 
   private final boolean showExtended;
+  private final boolean showDescription;
 
-  public ListStreams(final Optional<NodeLocation> location, final boolean showExtended) {
+  public ListStreams(
+      final Optional<NodeLocation> location,
+      final boolean showExtended,
```

Comment:
this is better modeled as an enum like:
```
// in it's own file, common to ListStreams/ListTables/ShowColumns
enum DescriptionType {
    DESCRIPTION,
    EXTENDED
}
```

admittedly minor point since I don't think it causes any issues, but this way you avoid invalid states (e.g. where both are true)

Reply:
Should `ShowColumns` use the enum even though `DESCRIBE` doesn't have the additional `DESCRIBE` option but just has regular and `EXTENDED`?